runtime/include/chpl-file-utils.h

```diff
@@ -40,6 +40,46 @@ qioerr chpl_fs_cwd(const char** working_dir);
 
 qioerr chpl_fs_exists(int* ret, const char* name);
 
+/*
+  Get the UID of the calling process. This function is UNIX only and cannot
+  fail.
+*/
+int chpl_fs_get_cuid(void);
+
+/*
+  Fetch a user's home directory given a username.
+
+  If a user given by the string "user" was able to be looked up in the password
+  database, this function will write a copy of that user's login directory to
+  "outs" and set "outf" to TRUE, else it will not write to "outs" and set
```

Comment:
There a potential mental parsing error in the else statement - I would change the last part of the sentence to "and *will* set ..." to avoid it